src/main/resources/locale_de.properties

```
@@ -252,30 +252,30 @@ command.roles.add.options.reputation.description=Die Menge an Reputation, die f
 command.roles.add.options.role.description=Die für die Reputation verwendete Rolle.
 command.roles.add.title.added=Rolle hinzugefügt.
 command.roles.description=Verwaltet die Reputationsrollen.
-command.roles.donor.add.description=Fügt eine Spender-Rolle hinzu, welche Reputation erhalten kann.
-command.roles.donor.add.message.add=Spender-Rolle %ROLE% hinzugefügt.
+command.roles.donor.add.description=Fügt eine Spenderrolle hinzu, welche Reputation erhalten kann.
```

I think this was wrong?
```suggestion
command.roles.donor.add.description=Fügt eine Spenderrolle hinzu, welche Reputation vergeben kann.
```